src/server/lib/app/application.ex

```diff
@@ -12,6 +12,7 @@ defmodule App.Application do
       App.Supervisor.Users,
       App.Auth,
       App.Process,
+      {Registry, keys: :unique, name: App.LobbyRegistry},
```

Comment:
when i implemented `App.Supervisor.Users`, the first time around i also used the `Registry`, but i realized it wasn't worth the complexity over using `:global`, which supports multi-node deployments as a bonus.